src/pinns_pde_solve.jl

```diff
@@ -658,7 +709,60 @@ function get_numeric_derivative()
         end
 end
 
+function get_numeric_integral(strategy, _indvars, _depvars, chain, derivative)
+    depvars,indvars,dict_indvars,dict_depvars = get_vars(_indvars, _depvars)
+    integral =
+        (u, cord, phi, integrating_var_id, integrand_func, lb, ub, θ ;strategy=strategy, indvars=indvars, depvars=depvars, dict_indvars=dict_indvars, dict_depvars=dict_depvars)->
+            begin
+                flat_θ = if (typeof(chain) <: AbstractVector) reduce(vcat,θ) else θ end
```

Comment:
why do you need it? θ should already be flat

Reply:
I checked it, it wasnt. I ll recheck